api/authentication/models.py

```diff
@@ -1,10 +1,16 @@
 import datetime
+import jwt
 
+from django.template.loader import render_to_string
 from django.core.mail import send_mail
 from django.db import models
 from django.utils import timezone
+from django.core.signing import TimestampSigner, b64_encode,b64_decode, BadSignature, SignatureExpired,force_bytes
+from django.core.mail import EmailMessage
 from django.contrib.auth.models import UserManager
-from django.contrib.auth.models import AbstractUser, Group
+from django.contrib.auth.models import AbstractBaseUser, AbstractUser, Group
+from settings import base
```

Comment:
dont use this import in future its bad practice
please use example in [docs](https://docs.djangoproject.com/en/2.2/topics/settings/#using-settings-in-python-code) `from django.conf import settings`